CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 2.8.12)
+if(WIN32)
+    cmake_minimum_required(VERSION 3.22.0)
+else()
+    cmake_minimum_required(VERSION 2.8.12)
+endif()
```

Comment:
Don't really like this but I suppose if the newer version is strictly required not much to do about it.

Reply:
Agreed, which is why it's only for Windows. Some of the fancier target tricks are definitely a no-go on CMake 2.8.12, and since this is a new port and I can only test on the latest, it makes sense to use that IMHO.